src/qt/recentrequeststablemodel.cpp

```diff
@@ -141,7 +142,7 @@ bool RecentRequestsTableModel::removeRows(int row, int count, const QModelIndex
         for (int i = 0; i < count; ++i)
         {
             const RecentRequestEntry* rec = &list[row+i];
-            if (!walletModel->saveReceiveRequest(rec->recipient.address.toStdString(), rec->id, ""))
+            if (!walletModel->wallet().saveReceiveRequest(DecodeDestination(rec->recipient.address.toStdString()), ToString(rec->id), ""))
```

Comment:
Should `#include <interfaces/wallet.h>`.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/18608#discussion_r407273795

> Should `#include <interfaces/wallet.h>`.

Thanks, updated